frontend/settings.js

```diff
@@ -0,0 +1,277 @@
+const { join, resolve } = require('path');
+
+// define global settings
+const globalSettings = {
+    // define the current context (root)
+    context: process.cwd(),
+
+    // build modes
+    modes: {
+        dev: 'development',
+        watch: 'development-watch',
+        prod: 'production',
+    },
+
+    // build modules
+    buildVariants: {
+        esm: 'esm',
+        legacy: 'legacy',
+    },
+
+    paths: {
+        // locate the typescript configuration json file
+        tsConfig: './tsconfig.yves.json',
+
+        // path to frontend build config json file
+        namespaceConfig: './config/Yves/frontend-build-config.json',
+
+        // core folders
+        core: './vendor/spryker/spryker-shop/Bundles',
```

Comment:
all settings paths should go to the project. if it copypasted from suite. You need to set the correct path to the settings file in stylelint. pls recheck all paths that you use in js

Reply:
There is a 'context' setting in this file, which defines the current root and is initialized as `process.cwd()` and all absolute paths in this file are calculated from this context. And all relative paths in node process are interpreted from the current working dir too. And current working dir is always /project. I tested the command using settings.js from suite-nonsplit without modification and it works as expected. And if it works fine, it will be simpler to maintain if it is unchanged, I think.

Reply:
pls make sure that is works correctly. You can try to remove this tool from suite -> create some error in FE and run this command. also modify files like `frontend-build-config.json` and other in suite. and make sure that this command is failed